### PR TITLE
chore: Format existing code with new formatter

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**
Formatted existing code with new formatter. Fixed existing issues, suppressed false positives.

APM-2639

### CODE REVIEW

**Comment by richard-bogdan-bitrise** [comment]
Note: branched from APM-2637

**Comment by richard-bogdan-bitrise** [comment]
This will look real long, sorry for it. The good news is that mainly it is just formatter updates because of indentation change. There are also some small fixes, some javadoc were missing line ending dots and similar issues.

**Review by daniellevass-bitrise — COMMENTED** [review]
Thanks for splitting this work in to separate commits - 53K/46K LOC change are way too much for a person to sensibly review - i had a look at a couple of the commits and the changes look sensible.

You mentioned in the comment there were a couple of "fixes" - would you be able to into a bit more detail what you mean by this? perhaps pulling them out individually with a comment so they're easier to find if to review?

**Comment by richard-bogdan-bitrise** [comment]
> Thanks for splitting this work in to separate commits - 53K/46K LOC change are way too much for a person to sensibly review - i had a look at a couple of the commits and the changes look sensible.
> 
> You mentioned in the comment there were a couple of "fixes" - would you be able to into a bit more detail what you mean by this? perhaps pulling them out individually with a comment so they're easier to find if to review?

Please use "Hide whitespace changes", it will help you in the review. Those fixes are manually removing/adding line breaks to javadoc (because they looked weird), adding some missing line ending dots, both will be highlighted with a different color. Note: in Android Studio there is an option to to Ignore whitespace and formatting changes (with the diff viewer), so it would make your review much easier.
Screenshot 2021-05-18 at 12 03 04

File: buildSrc/src/test/java/io/bitrise/trace/internal/ChangeLogHelperTest.java [hunk 1]
```diff
@@ -1,153 +1,163 @@
...
+    final List<ChangeLogEntry> changeLogEntries = Collections.emptyList();
+
+    final String actual = changeLogHelper.getReleaseName(lastTag, changeLogEntries);
+    assertTrue(actual.startsWith("### module2 - 0.1.1 -"));
+  }
+
+  @Test
+  public void getNewVersion_patchVersionIncrease() {
+    final String actual =
+        changeLogHelper.getNewVersion(dummyVersionName1, ChangeLogHelper.getPatchCommitTypes());
```
Comment (daniellevass-bitrise) [inline buildSrc/src/test/java/io/bitrise/trace/internal/ChangeLogHelperTest.java:83]: this maybe isn't the most optimal way to split the line? I presume it was auto generated and just picked the first place to line break?

**Review by daniellevass-bitrise — APPROVED** [review]
Not sure if some of the line breaks are the most sensible for readability, but I presume that was auto-generated and we had little control over that. Thank you for your work on this - it will really help make things easier to maintain in the future 🌟

**Comment by richard-bogdan-bitrise** [comment]
Rebased to resolve conflicts.

File: buildSrc/src/test/java/io/bitrise/trace/internal/ChangeLogHelperTest.java (hunk 1 above)
Comment (richard-bogdan-bitrise) [reply buildSrc/src/test/java/io/bitrise/trace/internal/ChangeLogHelperTest.java:83]: If I remove the line break, and do the auto format, the result is the same.

**Pull request merged**